tests/issues/128/sap-hana-preconfigure-issue-128-assert.yml

```diff
@@ -0,0 +1,49 @@
+- hosts: all
+  tasks:
+  - name: Get info about file /etc/init.d/boot.local
+    stat:
+      path: /etc/init.d/boot.local
+    register: stat_boot_local
+
+  - name: Assert that file /etc/init.d/boot.local exists
+    assert:
+      that: "stat_boot_local.stat.isreg == true"
+      fail_msg: "FAIL: The file /etc/init.d/boot.local does not exist or is no regular file!"
+      success_msg: "PASS: The file /etc/init.d/boot.local exists and is a regular file."
+    ignore_errors: "{{ sap_hana_preconfigure_assert_ignore_errors|d(false) }}"
+
+  - name: Assert that the mode of file /etc/init.d/boot.local is 0755
+    assert:
+      that: "stat_boot_local.stat.mode == '0755'"
+      fail_msg: "FAIL: The file /etc/init.d/boot.local has mode '{{ stat_boot_local.stat.mode }}' but the expected mode is '0755'!"
+      success_msg: "PASS: The file /etc/init.d/boot.local has mode 0755."
+    ignore_errors: "{{ sap_hana_preconfigure_assert_ignore_errors|d(false) }}"
```

Comment:
Skip this test if /etc/init.d/boot.local does not exist?  That's cleaner than just ignoring errors.

Reply:
I took over this code from the assertion code which I am using when running the role in assert mode, see branch bz1891642. I wanted to print a customized error message for each assertion. However, I forgot to remove the ignore_errors lines. These are not needed when running the tests.